_src/pldoc/static/js/tabs.js

```diff
@@ -0,0 +1,49 @@
+define([
+    'jquery'
+    ], function($) {
+
+    var Tabs = {
+
+        vars: {
+            tabContainer:   $('.pldoc-tab-wrapper'),
+            tabs:           $('.pldoc-tab-labels'),
+            tab:            $('.pldoc-tab-label'),
+            panels:         $('.pldoc-tabs'),
+            panel:          $('.pldoc-tab'),
+            activeClass:    'is-active',
+            hiddenClass:    'is-hidden',
+        },
+
+        init: function() {
+            Tabs.handleTabClick();
+        },
+
+        resetInterface: function(wrapper) {
+            wrapper.find('.pldoc-link').each(function() {
+                $(this).removeClass(Tabs.vars.activeClass);
```

Comment:
`$(this)` is the jQuery object in each iteration of the loop. The `event` being passed is the array key (i.e., 0, 1, 2, 3, etc.). What did you have in mind here?